app/routes/endemics/check-answers.js

```diff
@@ -584,14 +584,15 @@ const postHandler = {
         }
       }, request.logger)
 
-      setEndemicsClaim(request, 'reference', claim.reference)
-      setEndemicsClaim(request, 'amount', claim.data?.amount)
+      setEndemicsClaim(request, 'reference', claim.dataValues.reference)
```

Comment:
Does this need to be changed? The other end is not changing what it returns in response is it?

Reply:
I'm not hugely confident but I think it has been incorrect for a while now. It was setting the values using claim.reference and claim.data?.amount, but the 'claim' object which is returned from the application service is the DB Model, which has all the data contained inside dataValues.

Reply:
Fixed!